action.yml

```diff
@@ -29,27 +32,60 @@ outputs:
 runs:
   using: composite
   steps:
+    - id: registryconfig
+      if: ${{ cloudbees.registries != '' }}
```

Comment:
Since you have created an input for this, please update the condition to use `${{ inputs.registry-config }}`

Reply:
fixed